.storybook/preview.ts

```diff
@@ -0,0 +1,73 @@
+import type { Preview } from '@storybook/react'
+
+import { ThemeProvider, CssBaseline } from '@mui/material';
+import { withThemeFromJSXProvider } from '@storybook/addon-themes';
+
+import { lightTheme } from '../src';
+
+import '@fontsource/roboto/300.css';
+import '@fontsource/roboto/400.css';
+import '@fontsource/roboto/500.css';
+import '@fontsource/roboto/700.css';
+import '@fontsource/material-icons';
+
+// it seems that are a bug with the code generated by the docs with mui
+// it shows React.ForwardRef instead of the mui component name [bug]: https://github.com/storybookjs/storybook/issues/22287
+// the code below is a workaround on this, setting the displayName to the components seems to solve the problem
+import * as muiComps from '@mui/material'
```

Comment:
I think the name will appear if we define the components with as named function expressions:

```ts
forwardRef(function MyComponent() {} )
```

Reply:
I believe so, but in this case as mui seems to not being doing like this set the displayName seems to be the only way to fix the problem 😞